test/MHD/sphBragTDiffusion/python/testidefix.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr 27 14:00:41 2021
+
+@author: jmkempf
+"""
+
+import os
+import sys
+sys.path.append(os.getenv("IDEFIX_DIR"))
+from pytools.vtk_io import readVTK
+import numpy as np
+import inifix
+
+conf = inifix.load("../idefix.ini")
+amplitude = conf["Setup"]["amplitude"]
+gamma = conf["Hydro"]["gamma"]
+kappa = conf["Hydro"]["bragTDiffusion"][-1]
+time_step = conf["Output"]["vtk"]
+
+nfile = 6
+list_time = [round(i*time_step, 3) for i in range(nfile)]
+list_VTK = list()
+for k in range(nfile):
+    if k >= 10:
+        current_number = str(k)
+    else:
+        current_number = '0' + str(k)
+    current_VTK = readVTK('../data.00' + current_number + '.vtk', geometry='spherical')
+    list_VTK.append(current_VTK)
+
+list_VX = list()
+list_RHO = list()
+list_PRS = list()
+list_K = list()
+list_T = list()
+for VTK in list_VTK:
+    list_VX.append(VTK.data['VX1'])
+    list_RHO.append(VTK.data['RHO'])
+    list_PRS.append(VTK.data['PRS'])
+    list_K.append(VTK.data['PRS']/(VTK.data['RHO']**gamma))
+    list_T.append(VTK.data['PRS']/VTK.data['RHO'])
+R = VTK.r
```

Comment:
here you're relying on the fact that `for` loops in Python leak their iteration variables, let's not:
```suggestion
VTK = list_VTK[-1]
R = VTK.r
```